packages/amplify-graphql-http-transformer/src/graphql-http-transformer.ts

```diff
@@ -288,11 +324,23 @@ function createResolver(stack: cdk.Stack, dataSourceId: string, context: Transfo
           ref('util.qr($util.appendError($ctx.result.body, $ctx.result.statusCode))'),
         ),
       ),
+      `${functionId}.res.vtl`,
```

Comment:
nit: Can we use the naming convention of `TypeName.FieldName.DataSource.request/response.vtl`?
```suggestion
      `${config.resolverTypeName}.${config.resolverFieldName}.DataResolver.res.vtl`,
```